docs/cli/user-manual.md

```diff
@@ -1,293 +1,284 @@
+### Scenarios for `teamsfx new`
 
-### `--spfx-webpart-name`   
-Webpart Name. The default value is: "helloworld".
+Using interactive mode to create a Teams app is super intuitive, please try it by starting with `teamsfx new`. The following are the few scenerios on controlling all the parameters:
 
-### `--spfx-webpart-desp`
-Webpart Description. The default value is: "helloworld description".
+#### A tab app hosted on SPFx using React
 
-> Below options will take effect if `--host-type` is set to `azure`.
-
-### `--azure-resources`
-Add Azure resources to your project. Options(Multiple) are `sql` (Azure SQL Database) and `function` (Azure Functions).
-
-> Below Options will take effect if `--capabilities` include `bot` or `message-extension`.
-
-### `--way-to-register-bot`
-Options are `create-new` to Register a new bot or `reuse-existing` to reuse an existing one. Reuse existing bot requires `bot-id` and `bot-password`. The default value is: `create-new`.
-
-### `--bot-id`
-Bot id.
-
-### `--bot-password`          
-Bot password.
-    
-## Examples
-Using interactive mode to create a Teams app is super intuitive, please try it by starting with `teamsfx new`. Here are some examples for you to play with if you enjoy controlling all the parameters. 
-### A tab app hosted on SPFx using React
 ```bash
-teamsfx new --interactive false --app-name newspfxapp --host-type spfx --spfx-framework-type react
+teamsfx new --interactive false --app-name newspfxapp --capabilities tab-spfx --spfx-framework-type react
 ```
 
-### A Teams app contains multiple capabilities
+#### A Teams app in JavaScript contains tab, bot capabilities and Azure Functions
+
 ```bash
-teamsfx new --interactive false --app-name newtabbotapp --host-type azure --capabilities tab bot
+teamsfx new --interactive false --app-name newtabbotapp --capabilities tab bot --programming-language javascript --azure-resources function
 ```
 
-### A Teams tab app with Azure Functions and Azure SQL
+#### A Teams tab app with Azure Functions and Azure SQL
+
 ```bash
-teamsfx new --interactive false --app-name newapp --host-type azure --azure-resources sql function
+teamsfx new --interactive false app-name newapp --azure-resources sql function --programming-language typescript
 ```
 
-***
+## `teamsfx account`
 
-# teamsfx account
-Manage cloud service accounts. The supported cloud services are `azure` and `m365`.
+Manage cloud service accounts. The supported cloud services are `Azure` and `Microsoft 365`.
 
-## Commands
 | `teamsFx account` Commands  | Descriptions |
 |:----------------  |:-------------|
 | `teamsfx account login <service>`      | Log in to the selected cloud service. |
 | `teamsfx account logout <service>`      | log out of selected cloud service. |
-| `teamsfx account set`      | Update account settings. |
+| `teamsfx account set --subscription`      | Update account settings to set a subscription ID. |
 
```

Comment:
There may miss the `teamsfx account show` command.

Reply:
Thanks, added!